package.json

```diff
@@ -76,6 +76,7 @@
     "ember-source": "~3.28.8",
     "ember-template-lint": "^4.10.0",
     "ember-truth-helpers": "^3.0.0",
+    "ember-unique-id-helper-polyfill": "^1.1.0",
```

Comment:
Do we actually need this ?

Reply:
Not really, it's part of Ember in the 4.x release but you can just as easily swap it with hardcoded ids. The unique-id helper is mostly useful if you are dealing with dynamic form fields and want to ensure all the ids are unique.